lib/Stats.js

```diff
@@ -316,6 +316,9 @@ class Stats {
 			if (showErrorDetails && e.missing) {
 				text += e.missing.map(item => `\n[${item}]`).join("");
 			}
+			if (showErrorDetails && e.stack) {
```

Comment:
Some errors already include the stack trace in the `details`. Maybe it makes sense to only print the stack when no details are set.

```suggestion
			if (showErrorDetails && e.stack && !e.details) {
```

Reply:
The stack also include the `e.message` so avoid printing it twice in this case.

Reply:
Makes sense